### PR TITLE
Fix debugger breakpoints in the IOP JIT

### PR DESCRIPTION
**Pull request by Ziemas**
### Description of Changes
Switches from comparing directly to loading into a register and comparing

If the previous way was supposed to work I hope someone who understands this stuff can take a look at it.

### Rationale behind Changes
Fixes the debugger.

### Suggested Testing Steps
Make sure breakpoints work.

### CODE REVIEW

**Comment by kenshen112** [comment]
I'll test this evening

**Comment by refractionpcsx2** [comment]
hmm, is iopBreakpoint not aligned?

File: pcsx2/x86/iR3000A.cpp [hunk 1]
```diff
@@ -1206,7 +1206,8 @@ void psxRecMemcheck(u32 op, u32 bits, bool store)
 		next2.SetTarget();
 	}
 	// get out of here
-	xCMP(ptr8[&iopBreakpoint], 0);
+	xMOVZX(eax, ptr8[&iopBreakpoint]);
+	xCMP(eax, 0);
```
[the remark this answers is not in the document]
Comment (tellowkrinkle) [reply pcsx2/x86/iR3000A.cpp:1210]: Should be fixed by #5629

**Comment by Ziemas** [comment]
Confirmed fixed by #5629

**Pull request closed without merging**